README.md

```diff
@@ -109,10 +109,3 @@ We prefer that you squash commits into a single commit for a single change, mult
 From time to time LinkedIn maintainers may find issues with the changes that break our internal tests. When this happens we will
 allow the merge to happen but add tests and update the code to make our internal tests pass. This means that we probably won't
 publish a version until we can validate the changes internally.
-
-# Known Potential Issues
```

Comment:
Should we leave this so people who use old versions of pip know? @zvezdan what are your thoughts?